packages/veritone-react-common/src/components/LogoDetectionEngineOutput/index.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import classNames from 'classnames';
+import IconButton from 'material-ui/IconButton';
+import ZoomOutMap from 'material-ui-icons/ZoomOutMap';
+import Select from 'material-ui/Select';
+import { MenuItem } from 'material-ui/Menu';
+import EngineOutputHeader from '../EngineOutputHeader';
+import PillButton from '../Parts/Buttons/PillButton';
+import { msToReadableString } from '../../helpers/time'
+import styles from './styles.scss';
+
+export default class LogoDetectionEngineOutput extends Component {
+	static propTypes = {
+		label: PropTypes.string,
+		data: PropTypes.arrayOf(Object),
+		mediaPlayerTime: PropTypes.number,
+		className: PropTypes.string,
+		headerClassName: PropTypes.string,
+		bodyClassName: PropTypes.string,
+		itemClassName: PropTypes.string,
+		itemLabelClassName: PropTypes.string,
+		itemInfoClassName: PropTypes.string,
+		onItemSelected: PropTypes.func,
+		onEngineSelected: PropTypes.func,
+		onExpandViewClicked: PropTypes.func
+	};
+
+	static defaultProps = {
+		label: 'Logo Recognition',
+		data: [],
+		mediaPlayerTime: -1
+	};
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			isExpanded: false
+		};
+
+		this.onEngineChanged = this.onEngineChanged.bind(this);
```

Comment:
As discussed on Friday with @ChrisPelletier moving these selectors into a higher-level component keeps the props for this component much simpler and allows us to consolidate these selectors into a engine output viewer container component.

But if all other engine output components are currently doing this and we don't want to refactor them all right now, consistency wins.